lib/Compile.js

```diff
@@ -9,21 +9,113 @@ var Compile = module.exports = function (options) {
       
       var width = options.width;
       var jsonData = options.data;
+      
       var BASE_UNIT = options.baseUnit;
       
-      var Geometry = WorldBuilder.init(options.layers, BASE_UNIT);
-
       // TODO: THIS + 1 - 1 space odyssey has to stop
+      
+      if(options.bleedLayer) {
+        options.layers++; // For bleed edge
+        var startLength = jsonData.length-1;
+        
+        var bleedLayerLength = 139; // trixels
+        
+        for(var i=1; i<bleedLayerLength; i++) {
+          jsonData.push({
+            index: startLength + i,
+            color: "rgb(255,0,0)",
+            opacity: 1
+          });
+        }
+      }
+      
       var layersCount = options.layers + 1;
-
+      
+      var Geometry = WorldBuilder.init(options.layers, BASE_UNIT);
+      
       return createSVG(width);
-
+      
       /*======================= 
         
           utilities
             
       =======================*/ 
       
+      /*
```

Comment:
This utility function will be used for generalizing layers, it's irrelevant to bleedlayer